requirements.txt

```diff
@@ -0,0 +1,4 @@
+boto3
+click
+oauthlib
+requests
```

Comment:
To prevent future failures, as this script is not distributed as a library, maybe we should pin dependencies?